test/interpolation_tests.jl

```diff
@@ -35,10 +35,12 @@ function runtests()
 			nelement_local = nelement
 			nrank_per_block = 0 # dummy value
 			irank = 0 # dummy value
-			comm = MPI.COMM_NULL # dummy value 
+			comm = MPI.COMM_NULL # dummy value
+            element_spacing_option = "uniform"
```

Comment:
Could test sqrt spacing here. Change the `(discretization, rtol)` loop to `(discretization, element_spacing_option rtol)`...